### PR TITLE
Fix/75 bug on phone validation

### PR DESCRIPTION
**Pull request by ntchou**

### CODE REVIEW

File: src/components/Form/index.js [hunk 1]
```diff
@@ -64,53 +55,80 @@ const FormSubmit = styled.div`
...
+                                            disabled={isSubmitting}
+                                            title={t('phone.title')} 
+                                        />
+                                    </FormFields>
+                                </Tab>
+                                <Tab eventKey="mail" title={t('buttons.mail.label')}>
+                                    <FormFields>
+                                        <Field
+                                            name="mail"
+                                            type="mail"
```
Comment (HugoGresse) [inline src/components/Form/index.js:109]:
```suggestion
                                            type="email"
```

a "mail" is a letter in english

File: src/components/Form/validation.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    personName: Yup
+        .string()
+        .trim()
+        .required('personName.errors.required'),
+    contact: Yup.object().shape({
+        tab: 
+            Yup.ref('$tab'),
+        mail: Yup
+            .string()
+            .trim()
```
Comment (HugoGresse) [inline src/components/Form/validation.js:15]: trim() does not work when we submit the form with formik, you may consider using trim manually on submit

**Comment by ntchou** [comment]
> Can you also post a screenshot on the PR? It fasten the review

image

File: src/components/Form/index.js (hunk 1 above)
Comment (ntchou) [reply src/components/Form/index.js]: done

File: src/components/Form/validation.js (hunk 2 above)
Comment (ntchou) [reply src/components/Form/validation.js:15]: done

**Pull request merged**